src/screens/TopicPageScreen/index.js

```diff
@@ -50,22 +63,6 @@ const TopicPageScreen = (props) => {
   const [topicDetail, setTopicDetail] = React.useState({});
   const [memberCount, setMemberCount] = React.useState(0);
   const [isHeaderHide, setIsHeaderHide] = React.useState(false);
-  const opacityHeaderAnimation = React.useRef(new Animated.Value(1)).current;
-  const coverPath = topicDetail?.cover_path || null;
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
-
-  const animatedHeight = React.useRef(
-    new Animated.Value(
-      (coverPath
-        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-        dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-        topPosition +
-        normalize(4)
-    )
-  ).current;
-
   const [feedsContext, dispatch] = React.useContext(Context).feeds;
   const feeds = feedsContext.topicFeeds
     ? feedsContext.topicFeeds.filter((feed) => feed?.topics?.includes(topicName))
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `opacityHeaderAnimation` and `animatedHeight` could potentially impact the animation behavior of the header. Ensure that the new implementation provides the same or better user experience.
```diff
-  const opacityHeaderAnimation = React.useRef(new Animated.Value(1)).current;
-  const coverPath = topicDetail?.cover_path || null;
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
-  const animatedHeight = React.useRef(
-    new Animated.Value(
-      (coverPath
-        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
-        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) +
-        dimen.size.TOPIC_FEED_HEADER_HEIGHT +
-        topPosition +
-        normalize(4)
-    )
-  ).current;
+   const [topicDetail, setTopicDetail] = React.useState({});
+   const [memberCount, setMemberCount] = React.useState(0);
+   const [isHeaderHide, setIsHeaderHide] = React.useState(false);
+   const [feedsContext, dispatch] = React.useContext(Context).feeds;
+   const feeds = feedsContext.topicFeeds
+     ? feedsContext.topicFeeds.filter((feed) => feed?.topics?.includes(topicName))
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->